### PR TITLE
Project 2: Mobile Version

### PR DESCRIPTION
**Pull request by Paul-js-hub**
### Portfolio Project 2
In this milestone I:

- Added CSS for the about myself section
- Added CSS and Html codes for the work section
- Added styling to the work section
- Added CSS to the list of items

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,41 +1,217 @@
...
+                Html <span> <img src="images/image3.png" alt="html" /></span>
+              </li>
+              <li class="list-sumbua">
+                Bootstrap
+                <span> <img src="images/image3.png" alt="bootstrap" /></span>
+              </li>
+              <li>Ruby</li>
+            </ul>
+            <button class="seeproject-btn" type="button">See Project</button>
+          </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design)

hover

File: index.html [hunk 2]
```diff
@@ -1,41 +1,217 @@
...
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design)

hover

File: index.html [hunk 3]
```diff
@@ -1,41 +1,217 @@
...
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design)

hover

File: index.html [hunk 4]
```diff
@@ -1,41 +1,217 @@
...
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design)

hover

File: index.html (hunk 1 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] Kindly implement the spacing equally from both sides left and right to stick with the Figma design. Implement in all the section
- your design
his design

- Figma design
figma

File: index.html (hunk 3 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] Kindly stick as much as possible to Figma design. Implement the `Epplise` on all the sections in your project.

## Figma design

design

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Paul-js-hub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Paul-js-hub** [comment]
@roshan-bajgain does press mean to focus?

File: index.html (hunk 4 above)
Comment (Paul-js-hub) [reply index.html]: @roshan-bajgain is button pressed means to focus? Is enabled the one that is default

File: index.html (hunk 3 above)
Comment (Paul-js-hub) [reply index.html]: @roshan-bajgain what do you mean by is a link a or button tag? I included a button tag as you can see in the HTML codes

**Review by rukundoeric — APPROVED** [review]
Hi @Paul-js-hub,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**